staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -165,6 +169,7 @@ func (d *validatingDispatcher) callHook(ctx context.Context, h *v1beta1.Validati
 	if err := r.Do().Into(response); err != nil {
 		return &webhookutil.ErrCallingWebhook{WebhookName: h.Name, Reason: err}
 	}
+	trace.Step("Request completed")
```

Comment:
I believe we do allow validation webhooks to be concurrent. So it seems like we need this request completed to indicate which validating webhook call it corresponds to.

Reply:
See https://github.com/kubernetes/kubernetes/pull/78610#discussion_r292639737